masonry/src/contexts.rs

```diff
@@ -627,18 +608,35 @@ impl_context_method!(
 pub struct TimerToken;
 
 impl EventCtx<'_> {
+    // TODO - Document
+    // TODO - Figure out cases where widget should be notified of pointer capture
+    // loss
+    // TODO - Panic when event isn't PointerDown
```

Comment:
I wonder whether this should probably be done in this PR? It would make reviewing easier at least. (Same for the other methods)

Reply:
I've added a quick-and-dirty assertion. Documentation will be for a later PR.